src/io/keyboard.js

```diff
@@ -1,11 +1,16 @@
 var Cast = require('../util/cast');
 
-function Keyboard () {
+function Keyboard (runtime) {
     /**
      * List of currently pressed keys.
      * @type{Array.<number>}
      */
     this._keysPressed = [];
+    /**
+     * Reference to the owning Runtime. Can be used, for example, to activate hats.
```

Comment:
Looks like the test is actually failing because this line is too long. If we split it into two, it'll be good.

P.S. you may want to set up ESLint in your text editor so it tells you immediately.